src/stages/main/patchers/YieldFromPatcher.js

```diff
@@ -0,0 +1,22 @@
+import NodePatcher from './../../../patchers/NodePatcher.js';
+import type { Node, ParseContext, Editor } from './../../../patchers/types.js';
+
+export default class YieldFromPatcher extends NodePatcher {
+  expression: NodePatcher;
+  
+  constructor(node: Node, context: ParseContext, editor: Editor, expression: NodePatcher) {
+    super(node, context, editor);
+    this.expression = expression;
+  }
+
+  /**
+   * YIELD FROM EXPRESSION
+   */
+  patchAsExpression({ needsParens=true }={}) {
+    let src = this.sourceTokenAtIndex(this.contentStartTokenIndex);
```

Comment:
Well, actually not, I don't understand to much about the code, but in the test contentStart/contentEnd also overwrites the (elsewhere) added `return` statement, so I had to go through tokens